ldes-server-port-ingest-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/rest/validators/pathsingestvalidator/PathsIngestValidatorTest.java

```diff
@@ -0,0 +1,80 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.ingest.rest.validators.pathsingestvalidator;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.events.admin.EventStreamCreatedEvent;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.EventStream;
+import be.vlaanderen.informatievlaanderen.ldes.server.ingest.rest.exception.IngestValidationException;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.RDFDataMgr;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+import java.net.URISyntaxException;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PathsIngestValidatorTest {
+    private final String TIMESTAMP_PATH = "http://purl.org/dc/terms/created";
+    private final String VERSIONOF_PATH = "http://purl.org/dc/terms/isVersionOf";
+    private PathsIngestValidator validator;
+
+    @BeforeEach
+    void setup() {
+        validator = new PathsIngestValidator();
+        validator.handleEventStreamInitEvent(new EventStreamCreatedEvent(
+                new EventStream("state", TIMESTAMP_PATH, VERSIONOF_PATH, true)));
+        validator.handleEventStreamInitEvent(new EventStreamCreatedEvent(
+                new EventStream("version",TIMESTAMP_PATH, VERSIONOF_PATH, false)));
+    }
+
+    @ParameterizedTest
+    @ArgumentsSource(IncorrectMemberArgumentsProvider.class)
+    void when_IncorrectMemberReceived_Then_ValidationThrowsException(Model model, String collectionName) {
+        assertThrows(IngestValidationException.class, () -> validator.validate(model, collectionName));
```

Comment:
Can the error message be tested as well, to verify what the validation violation is? This could have detected the mentioned bug above